src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -476,24 +476,28 @@ static inline int MPIDI_NM_mpi_put(const void *origin_addr,
                                    int target_count, MPI_Datatype target_datatype, MPIR_Win * win,
                                    MPIDI_av_entry_t * addr)
 {
-    int mpi_errno = MPI_SUCCESS;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_MPI_PUT);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_PUT);
+    int mpi_errno = MPI_SUCCESS;
```

Comment:
Declaration after statement.

Reply:
I don't understand your comment.  Is this change making it consistent?

Reply:
Sorry about the confusion. No, this change does not make the code consistent but actually make it inconsistent. However, because this commit needs to revert another commit, I think it has to change everything of that commit back. After reverting, we could re-do the correct consistency changes, including the one we discuss here.

Reply:
Oh, the revert commit was throwing me off.  Nevermind.  I looked at the individual commits now and it makes sense.  I'll approve it.  Sorry for the noise.